core/jazz_cloud-logs-streamer/_ES/apilogs.json

```diff
@@ -6,25 +6,25 @@
   "mappings" : {
     "apilogs" : {
        "properties" : {
-			"timestamp" : { "type":"date","format":"dateOptionalTime" },
-			"platform_log_group" : { "type":"string","index":"not_analyzed" },
-			"platform_log_stream" : { "type":"string","index":"not_analyzed" },
-			"environment" : { "type": "string", "index": "not_analyzed" },
-			"request_id" : { "type": "string", "index": "not_analyzed" },
-			"method" : { "type": "string", "index": "not_analyzed" },
-			"domain" : { "type": "string", "index": "not_analyzed" },
-			"servicename" : { "type": "string", "index": "not_analyzed" },
-			"path" : { "type": "string", "index": "not_analyzed" },
-			"user_agent" : { "type": "string", "index": "not_analyzed" },
-			"application_logs_id" : { "type": "string", "index": "not_analyzed" },
-			"origin" : { "type": "string", "index": "not_analyzed" },
-			"host" : { "type": "string", "index": "not_analyzed" },
-			"x_forwarded_port" : { "type": "string", "index": "not_analyzed" },
-			"x_forwarded_for" : { "type": "string", "index": "not_analyzed" },
-			"x_amzn_trace_id" : { "type": "string", "index": "not_analyzed" },
-			"content_type" : { "type": "string", "index": "not_analyzed" },
-			"cache_control" : { "type": "string", "index": "not_analyzed" },
-			"status" : { "type": "string", "index": "not_analyzed" }
+                        "timestamp" : { "type":"date","format":"dateOptionalTime" },
+                        "platform_log_group" : { "type":"keyword","index":"false" },
```

Comment:
I think `index` should be `true` according to this [guide](https://www.elastic.co/blog/strings-are-dead-long-live-strings).

Reply:
Done